### PR TITLE
[UX2.0] Adds Policy Objects Profile Parcels - Part 1

### PR DESCRIPTION
**Pull request by seconroy**
### Description

To test the `TF_VAR_policy_object_feature_template_id` must be configured, For example:
```
export TF_VAR_policy_object_feature_template_id=e4d9392a-7765-4a64-b719-a4bcaf534f25
```

Also adds the following Policy Objects profile parcel resource and data source:
- class
- tloc
- color
- data-ipv6-prefix
- data-prefix
- expanded-community
- ext-community
- ipv6-prefix
- prefix
- mirror
- policer

### Types of Changes

* [x] New feature (non-breaking change which adds functionaality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Build/CI change
* [ ] Code quality improvement/refactoring/documentation (no functional changes)

### Checklist

* [x] My code follows the code style of this project
* [x] I have added tests to cover my changes
* [x] All new and existing tests pass locally

### CODE REVIEW

File: gen/definitions/profile_parcels/policy_object_color.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: Policy Object Color
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_color.yaml:2]: Rename to "Policy Object Color List"

File: gen/definitions/profile_parcels/policy_object_data_ipv6_prefix.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+name: Policy Object Data IPv6 Prefix
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_data_ipv6_prefix.yaml:2]: Rename to "Policy Object Data IPv6 Prefix List"

File: gen/definitions/profile_parcels/policy_object_data_prefix.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+name: Policy Object Data Prefix
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_data_prefix.yaml:2]: Rename to "Policy Object Data IPv4 Prefix List"

File: gen/definitions/profile_parcels/policy_object_expanded_community.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+name: Policy Object Expanded Community
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_expanded_community.yaml:2]: Rename to "Policy Object Expanded Community List"

File: gen/definitions/profile_parcels/policy_object_extended_community.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+name: Policy Object Extended Community
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_extended_community.yaml:2]: Rename to "Policy Object Extended Community List"

File: gen/definitions/profile_parcels/policy_object_ipv6_prefix.yaml [hunk 6]
```diff
@@ -0,0 +1,30 @@
+---
+name: Policy Object IPv6 Prefix
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_ipv6_prefix.yaml:2]: Rename to "Policy Object IPv6 Prefix List"

File: gen/definitions/profile_parcels/policy_object_prefix.yaml [hunk 7]
```diff
@@ -0,0 +1,30 @@
+---
+name: Policy Object Prefix
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_prefix.yaml:2]: Rename to "Policy Object IPv4 Prefix List"

File: gen/definitions/profile_parcels/policy_object_tloc.yaml [hunk 8]
```diff
@@ -0,0 +1,32 @@
+---
+name: Policy Object TLOC
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_tloc.yaml:2]: Rename to "Policy Object TLOC List"

File: gen/definitions/profile_parcels/policy_object_ipv6_prefix.yaml [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+      id: true
+      mandatory: true
+      example: 2001:db8:85a3::8a2e:370:7334
+      minimum_test_value: '"2001:db8:85a3::8a2e:370:7334"'
+    - model_name: ipv6PrefixLength
+      id: true
+      mandatory: true
+      example: 64
+      minimum_test_value: '64'
+    - model_name: leRangePrefixLength
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_ipv6_prefix.yaml:27]: should we change it to just "le" to match what we've had for UX1.0 ipv6 prefix-list?

File: gen/definitions/profile_parcels/policy_object_ipv6_prefix.yaml [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+      example: 100
+    - model_name: geRangePrefixLength
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_ipv6_prefix.yaml:29]: should we change it to just "ge" to match what we've had for UX1.0 ipv6 prefix-list?

File: gen/definitions/profile_parcels/policy_object_prefix.yaml [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+      id: true
+      mandatory: true
+      example: 10.0.0.0
+      minimum_test_value: '"10.0.0.0"'
+    - model_name: ipv4PrefixLength
+      id: true
+      mandatory: true
+      example: 8
+      minimum_test_value: '8'
+    - model_name: leRangePrefixLength
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_prefix.yaml:27]: should we change it to just "le" to match what we've had for UX1.0 ipv4 prefix-list?

File: gen/definitions/profile_parcels/policy_object_prefix.yaml [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+      example: 24
+    - model_name: geRangePrefixLength
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_prefix.yaml:29]: should we change it to just "ge" to match what we've had for UX1.0 ipv4 prefix-list?

File: gen/definitions/profile_parcels/policy_object_tloc.yaml [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_env_name: POLICY_OBJECT_FEATURE_TEMPLATE_ID
+  - model_name: entries
+    mandatory: true
+    attributes:
+    - model_name: tloc
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_tloc.yaml:17]: should we add "tf_name: tloc_ip" to match what we've had for UX1.0 tloc-list?

File: gen/definitions/profile_parcels/policy_object_policer.yaml [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+    mandatory: true
+    attributes:
+    - model_name: burst
+      tf_name: burst_bytes
+      id: true
+      mandatory: true
+      example: 56500
+      minimum_test_value: '56500'
+    - model_name: exceed
+      tf_name: select_value
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_policer.yaml:24]: seems not correct. Should we change "tf_name: select_value" to "tf_name: exceed_action"?

**Review by tzarski0 — CHANGES_REQUESTED** [review]

**Review by danischm — APPROVED** [review]

**Pull request merged**